src/app.js

```diff
@@ -1,12 +1,26 @@
 const express = require('express');
 const path = require('path');
-// const exphbs = require('express-handlebars');
-// const controllers = require('./controllers/');
-
+const exphbs = require('express-handlebars');
+const controllers = require('./controllers/');
 
 const app = express();
 
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'hbs');
+app.engine(
+  'hbs',
+  exphbs({
+    extname: 'hbs',
+    layoutsDir: path.join(__dirname, 'views', 'layouts'),
+    partialsDir: path.join(__dirname, 'views', 'partials'),
+    defaultLayout: 'main',
+  }),
+);
+
+app.set('port', process.env.PORT || 3000);
```

Comment:
port is being set in index.js - so not needed here? we should do one or the other, not both.

Reply:
Good point, fixing it now